unittests/runtime/CMakeLists.txt

```diff
@@ -88,9 +88,9 @@ if(("${SWIFT_HOST_VARIANT_SDK}" STREQUAL "${SWIFT_PRIMARY_VARIANT_SDK}") AND
 #    list(APPEND PLATFORM_SOURCES
 #      DistributedActor.cpp
 #      )
-    list(APPEND PLATFORM_TARGET_LINK_LIBRARIES
-      swift_Distributed${SWIFT_PRIMARY_VARIANT_SUFFIX}
-      )
+#    list(APPEND PLATFORM_TARGET_LINK_LIBRARIES
+#      swift_Distributed${SWIFT_PRIMARY_VARIANT_SUFFIX}
```

Comment:
Whats up with this here?

Reply:
I think I can revert this. It caused some problems initially because the RPATH was not set correctly.